packages/frontend/.env.example

```diff
@@ -1 +1,2 @@
 REACT_APP_WALLET_CONNECT_PROJECT_ID=
+PRIVATE_KEY= # For Synpress E2E testing only
```

Comment:
Name is a big generic.

Reply:
@boriskubrik As required by synpress, not exposed in the config as far as I could tell